bangazonapi/models/product.py

```diff
@@ -62,9 +62,16 @@ def average_rating(self):
         for rating in ratings:
             total_rating += rating.rating
 
-        avg = total_rating / len(ratings)
+        if total_rating > 0:
+
+            avg = total_rating / len(ratings)
```

Comment:
You're close! The ticket #15 asks for a `try..except` block here, rather than an `if..else`. See if you can apply a `try..except` block to handle this `ZeroDivisionError`!